test/blackbox-tests/test-cases/pkg/ocamlformat/binary-avoid-from-project-deps.t

```diff
@@ -0,0 +1,174 @@
+If the dev-tool feature is enabled then `dune fmt` should invoke the `ocamlformat`
+executable from the dev-tool and not the one from the project's regular package
+dependencies.
+
+If the dev-tool feature is not enabled then `dune fmt` should invoke the
+`ocamlformat` executable from the project's regular package dependencies.
+
+  $ . ../helpers.sh
+  $ mkrepo
+
+Make a fake ocamlformat.0.26.2:
+  $ mkdir ocamlformat
+  $ cd ocamlformat
+  $ cat > dune-project <<EOF
+  > (lang dune 3.13)
+  > (package (name ocamlformat))
+  > EOF
+  $ cat > ocamlformat.ml <<EOF
+  > let version = "0.26.2"
+  > ;;
+  > let () = print_endline ("formatted with version "^version)
+  > EOF
+  $ cat > dune <<EOF
+  > (executable
+  >  (public_name ocamlformat))
+  > EOF
+  $ cd ..
+  $ tar -czf ocamlformat-0.26.2.tar.gz ocamlformat
+
+Make a fake ocamlformat.0.26.3:
```

Comment:
I see these written both like OCamlFormat 0.26.3 and ocamlformat.0.26.3. If the latter is correct, it will need to be in monospace `ocamlformat.0.26.3`. 

Let me know which format is correct, and I'll go through and make them all consistent.

Reply:
It is `ocamlformat.0.26.3`.

Reply:
Thanks for the response! To clarify, you want it written in monospace or without? I believe the correct way would be in monospace since it appears in the code, no? Let me know and I'll make all of them consistent.

Reply:
Hi Christine, The backquotes syntax for code is only for files that are in the markdown format. Our tests are not written in markdown, so the backticks aren't needed.

Reply:
Ah! Thank you for that explanation. Some of them are in .ml, .mli. and .t, all which have their own way to put things in code block. Shall I do that?

If you don't want the things normally in monospace formatted that way in these (code, libraries, specific files, etc.), how do you want them formatted to indicate what they are? I saw a lot of them were in " " -- but not all of them. Whichever you'd like, please let me know so I can make all of them consistent.